create_table.lua

```diff
@@ -1,8 +1,13 @@
 box.cfg{listen = 3301, memtx_memory = 10 * 1024^3}
 box.schema.user.grant('guest', 'read,write,execute', 'universe')
 
-box.sql.execute("drop table if exists warehouse;")
-box.sql.execute("create table warehouse ( \
+-- Prior to 2.2.1 box.sql.execute was used for SQL queries.
+if not box.execute then
+	box.execute = box.sql.execute
+end
```

Comment:
Nit: Please, use four whitespaces indentation in a Lua code instead of tabs.

Reply:
Done